docs/xenserver-automated-certification-kit-guide.md

```diff
@@ -271,19 +273,19 @@ See below for a description of the keys:
 - **network_id** - ID of the logical network on layer 2
 - **vlan_ids** - VLAN ID used in the test, delimited by comma if multiple IDs present.
 - **vf_driver_name** - Driver name of the SR-IOV VF to be used by the Droid VM
-  If the key is specified, ACK will write the value, denoted as `<name>`, into file /etc/modulesload.d/`<name>`.conf of the Droid VM. Then the VM is able to load it automatically during booting.If the key is not specified or its value is empty, it indicates that the VF driver is already able to be loaded automatically.
+  If the key is specified, server certification kit will write the value, denoted as `<name>`, into file /etc/modulesload.d/`<name>`.conf of the Droid VM. Then the VM is able to load it automatically during booting.If the key is not specified or its value is empty, it indicates that the VF driver is already able to be loaded automatically.
 - **vf_driver_pkg** -  Name of the driver package (.rpm) for the SR-IOV VF. The package is located under /opt/xensource/packages/files/auto-cert-kit/ of the  pool coordinator host.
-If the key is specified, ACK will upload the specified driver to the Droid VM and install.
+If the key is specified, server certification kit will upload the specified driver to the Droid VM and install.
 If the key is not specified or its value is empty, it indicates the driver has already been installed in the Droid VM.
 
 
 > **Note**:
 >
-> The Droid VM used by ACK is /opt/xensource/packages/files/auto-cert-kit/vpxdlvm.xva, which is based on upstream CentOS 7.2 X86_64 and kernel version is 3.10.0. The specified .rpm package must be applicable to the Droid VM in use.
+> The Droid VM used by server certification kitis is /opt/xensource/packages/files/auto-cert-kit/vpxdlvm.xva, which is based on upstream CentOS 7.2 X86_64 and kernel version is 3.10.0. The specified .rpm package must be applicable to the Droid VM in use.

```

Comment:
`kitis`

Reply:
We already changed to ask user to install Rocky, so need to update all about `CentOS`

Reply:
Has correct it, thanks